ft-fine-tune-distilgpt2/fine-tune-unittests.py

```diff
@@ -75,16 +66,20 @@ def test_return_metrics(self):
         model_name = "distilgpt2"
         output_model_path = "story_generator_checkpoint_"
         distilgpt2_target = 90
-        
-        cmd = "python -u ./finetune.py --model_name distilgpt2 "
+        finetune_path = os.path.join(scripts_dir, 'finetune.py')
+
+        cmd = "python -u " + finetune_path + " --model_name distilgpt2 " \
```

Comment:
I do not see a setup function in which you are reading test parameters? It looks you are still hard coding this test params! 

What about the files path fix I mentioned in my previous review.  Is this implemented some place?

Reply:
Basem, Thanks for your comments. I re-submitted the code for your review by parsing testing parameters from unittests as well. Originally I parsing testing parameters in main script by using library yamls. 

In addition, I fixed paths by using absolute paths in each function as below: 
scripts_dir = pl.Path(__file__).parent.resolve()
finetune_path = os.path.join(scripts_dir, 'finetune.py')